### DISCUSSION TITLE
Is there a BSidebar component?

### DISCUSSION THREAD

**Discussion by jacobg** [cdmoro/bootstrap-vue-3#1832] [General]
bootsrap-vue version 2 has one:
https://github.com/bootstrap-vue/bootstrap-vue/tree/dev/src/components/sidebar

**Comment by VividLemon** [comment]
https://getbootstrap.com/docs/5.3/components/offcanvas/ & https://bootstrap-vue-next.github.io/bootstrap-vue-next/docs/components/offcanvas.html
